frontend/src/pages/Signup.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import '../components/Styles/Signup.css'
+import axios from "axios";
+import { createRoot } from "react-dom/client";
+
+
+const Signup = () => {
+
+//   const [userid, setUserid] = useState("");
+//   const [firstname, setFirstname] = useState("");
+//   const [lastname, setLastname] = useState("");
+//   const [username, setUsername] = useState("");
+//   const [password, setPassword] = useState("");
+//   const [birthday, setBirthday] = useState("");
+//   const [age, setAge] = useState("");
+//   const [gender, setGender] = useState("");
+
+//   const handleClick = () => {
+//     axios.post('https://localhost:8800/signup',
+//     {
+//         userid: userid,
+//         firstname: firstname,
+//         lastname: lastname,
+//         username: username,
+//         password: password,
+//         birthday: birthday,
+//         age: age,
+//         gender: gender
+//     }
+//   )
+//     .then((response) => {
+//         console.log(response);
+//     });
+const [formData, setFormData] = useState({
+    userid: 0,
+    Fname: "",
+    Lname: "",
+    username: "",
+    password: "",
+    Bday: new Date(), 
+    age: 0,
+    gender: "",
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+  
+    // Check if the field is 'Bday' and parse the value into a Date object
+    const updatedValue = name === 'Bday' ? new Date(value) : value;
+  
+    setFormData((prevFormData) => ({
+      ...prevFormData,
+      [name]: updatedValue,
+    }));
+  };
+
+  const formatDate = (date) => {
+    // Format the date as needed, for example: YYYY-MM-DD
+    return date.toISOString().split('T')[0];
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
```

Comment:
after submitting the user's data, where does it go from here?
Does user have to click something to go somewhere
 or it automatically go to after signing up?